README.md

```diff
@@ -24,6 +24,9 @@ $ git checkout -b <new-branch-name>
 5. You’ll get your bot API token as `89xxxxxxx:xxxxxxxxxxxxAAHmf32ZghS-cqxBLfnkUx9VwoXeOIRlnUQ`.
 6. Copy it and save it in a file named as `/.env/` in your project folder.
 
+## Set Mongo DB
+1. The configuration of your mongo db should be in : mongodb://localhost:27017
+
```

Comment:
Thanks for pointing this out. Proper setup instructions are already listed in https://github.com/osdc/bots/blob/master/CONTRIBUTING.md. We will update the README as part of #48 so it's not needed in this PR :)